pkg/storage/unified/search/bleve_mappings.go

```diff
@@ -0,0 +1,94 @@
+package search
+
+import (
+	"github.com/blevesearch/bleve/v2"
+	"github.com/blevesearch/bleve/v2/analysis/analyzer/keyword"
+	"github.com/blevesearch/bleve/v2/mapping"
+
+	"github.com/grafana/grafana/pkg/storage/unified/resource"
+)
+
+type bleveFlatDocument struct {
+	// The group/resource (essentially kind)
+	GR string `json:"gr,omitempty"`
+
+	Title       string `json:"title,omitempty"`
+	TitleSort   string `json:"title_sort,omitempty"`
```

Comment:
we duplicate this field because the index requirements for search vs sort are different